src/fields/index.js

```diff
@@ -12,5 +13,6 @@ module.exports = {
   ChoiceGroup: ChoiceGroup,
   Image: Image,
   Text: Text,
-  TextArea: TextArea
+  TextArea: TextArea,
+  addCounter: addCounter
```

Comment:
Exposing this HOC as if it's a field type feels like the wrong thing to do. If we want to expose it, we should explain that people can use it by doing something like `const addCounter = require('react-formbuilder/addcounter`), keeping it its own file (since a HOC is not itself a field type, it's just a utility wrapper).

Reply:
Should i move it outside fields directory?

Reply:
I would say for now to just leave it off, and then we can file a completely separate issue for exposing just this HOC on its own somewhere in future work.